README.md

```diff
@@ -1,8 +1,23 @@
 # Kleros Interaction Smart Contracts
 
+<p align="center">
+  <b style="font-size: 32px;">Kleros</b>
+</p>
+
+<p align="center">
+  <a href="https://standardjs.com"><img src="https://img.shields.io/badge/code_style-standard-brightgreen.svg" alt="JavaScript Style Guide"></a>
+  <a href="https://travis-ci.org/kleros/kleros"><img src="https://travis-ci.org/kleros/kleros.svg?branch=master" alt="Build Status"></a>
+  <a href="https://david-dm.org/kleros/kleros"><img src="https://david-dm.org/kleros/kleros.svg" alt="Dependencies"></a>
+  <a href="https://david-dm.org/kleros/kleros?type=dev"><img src="https://david-dm.org/kleros/kleros/dev-status.svg" alt="Dev Dependencies"></a>
```

Comment:
These are the badges for `kleros/kleros`.

Reply:
Ooops, yeah.